qcelemental/models/basis.py

```diff
@@ -95,18 +110,58 @@ class BasisSet(ProtoModel):
     schema_name: constr(strip_whitespace=True, regex="qcschema_basis") = "qcschema_basis"
     schema_version: int = 1
 
-    basis_name: str = Schema(..., description="A standard basis name if available (e.g., 'cc-pVDZ'.")
+    name: str = Schema(..., description="A standard basis name if available (e.g., 'cc-pVDZ'.")
     description: Optional[str] = Schema(None, description="A brief description of the basis set.")
-    basis_data: Dict[str, BasisCenter] = Schema(..., description="A mapping of all types of centers available.")
-    basis_atom_map: List[str] = Schema(
-        ..., description="Mapping of all centers in the parent molecule to centers in `basis_data`.")
+    center_data: Dict[str, BasisCenter] = Schema(..., description="A mapping of all types of centers available.")
+    atom_map: List[str] = Schema(
+        ..., description="Mapping of all centers in the parent molecule to centers in `center_data`.")
+
+    nbf: Optional[int] = Schema(None, description="The number of basis functions.")
```

Comment:
should BasisSet schema_version bump to 2 with these renamings so that easily differentiate from Ben's records?

Reply:
I don't know if the BasisSet schema was ever fully released. It would agree in principle however.